src/process/routes/index.js

```diff
@@ -31,6 +31,5 @@ const Project = {
   path: Paths.PROJECT_PATH
 };
 
-const routes = [CreateProject, EditProject, Explore, Home, MyProjects, Project];
-
-export default routes;
+export const publicRoutes = [Explore, Home, Project];
+export const privateRoutes = [CreateProject, EditProject, MyProjects];
```

Comment:
```suggestion
const publicRoutes = [Explore, Home, Project];
const privateRoutes = [CreateProject, EditProject, MyProjects];

export { publicRoutes, privateRoutes }
```


also in /paths.js i want you to change how the export is done like this

```
const DASHBOARD_PATH = '/dashboard';
const EVENTS_PATH = '/events';
const FORGOT_PASSWORD_PATH = '/forgot-password';
const INBITES_PATH = '/inbites';
const INFLUENCERS_PATH = '/influencers';
const NOTIFICATIONS_PATH = '/notifications';
const RESET_PASSWORD_PATH = '/reset-password';
const RESET_PASSWORD_RESTRICTION_PATH = '/reset-password-restriction';
const SIGN_IN_PATH = '/';
const SIGN_UP_PATH = '/signup';

const DEFAULT_PRIVATE_PATH = DASHBOARD_PATH;
const DEFAULT_PUBLIC_PATH = SIGN_IN_PATH;

const privatePaths = {
  DASHBOARD_PATH,
  DEFAULT_PATH: DEFAULT_PRIVATE_PATH,
  EVENTS_PATH,
  INBITES_PATH,
  INFLUENCERS_PATH,
  NOTIFICATIONS_PATH
};

const publicPaths = {
  DEFAULT_PATH: DEFAULT_PUBLIC_PATH,
  FORGOT_PASSWORD_PATH,
  RESET_PASSWORD_PATH,
  RESET_PASSWORD_RESTRICTION_PATH,
  SIGN_IN_PATH,
  SIGN_UP_PATH
};

export default { public: publicPaths, private: privatePaths };

```
so when you want to use paths you'll import all of them like `import Paths from 'process/routes/paths` and use them like `Paths.public.MY_PROJECTS_PATH`

Reply:
cam greu